static/fluent/en-US/main.ftl

```diff
@@ -1,9 +1,22 @@
+## The following terms must be treated as brand, and kept in English.
+##
+## They cannot be:
+## - Transliterated.
+## - Translated.
+##
+## Declension should be avoided where possible.
+##
+## Reference: https://mozilla-l10n.github.io/styleguides/mozilla_general/index.html#brands-copyright-and-trademark
+
+-webthings-gateway-brand = WebThings Gateway
+-mozilla-iot-brand = Mozilla IoT
+
 # Main Title
-webthings-gateway = WebThings Gateway
+webthings-gateway = { -webthings-gateway-brand }
 
 # Wordmark
 wordmark =
-  .alt = Mozilla IoT
+  .alt = { -mozilla-iot-brand }
```

Comment:
Where in the UI is this being used?

Reply:
It's the `alt` text for the wordmark image.

Reply:
I suppose we should just be using the WebThings Gateway brand.

Reply:
Yeah I think this should have been changed when the logo was changed

Reply:
(probably my bad)

Reply:
Fixed.